### DISCUSSION TITLE
Badges/Shields

### DISCUSSION THREAD

**Discussion by mfhodges** [community/community#21912] [New to GitHub]
<p>Hi,</p>
<p>I have been using GitHub for some time, but I’ve always been a little confused by the meaning of badges/shields on repo’s readme. I know they offer some metadata about the repo, so I guess my question is more philosophical about when and why to use them.</p>
<p>Thanks!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>They’re a nice quick reference for items that matter to some groups or to reflect the status of the repo at a given time. For example:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/ae91cc32a81853f022ea742095f5b44319a78637.png" data-download-href="/uploads/short-url/oUjrTkAcJ0gbc6jtc9f69s7RfXF.png?dl=1" title="Screen Shot 2018-02-26 at 4.54.55 PM.png"><img src="https://user-images.githubusercontent.com/1038121/181067506-4a6564d8-c010-4b43-823b-4329d59b3abb.png" alt="Screen Shot 2018-02-26 at 4.54.55 PM.png" data-base62-sha1="oUjrTkAcJ0gbc6jtc9f69s7RfXF" width="690" height="106" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/ae91cc32a81853f022ea742095f5b44319a78637_2_690x106.png, https://aws1.discourse-cdn.com/github/original/2X/a/ae91cc32a81853f022ea742095f5b44319a78637.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/a/ae91cc32a81853f022ea742095f5b44319a78637.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/ae91cc32a81853f022ea742095f5b44319a78637_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-02-26 at 4.54.55 PM.png</span><span class="informations">890×138 12.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>This is from one of my repos. The first is the latest version published to the package manager (in case I forget to update the docs). The second is the link to the docs. The third I believe I added before GitHub added the auto-detecting License tab in the repo header. And the fourth is a status of how “clean” the code is. None of them are terribly important really … but it is a nice quick-reference. It is a compact way to deliver a bunch of tidbits of information about the current status of the project.</p>
<p>I’m not sure if that clears anything up but I hope that helps!</p>
